doc-content/enterprise-only/processes/interacting-with-processes-tasks-special-filter-proc.adoc

```diff
@@ -0,0 +1,17 @@
+[id='interacting-with-processes-tasks-special-filter-proc']
+= Defining special filters for tasks
+
+You can create special filters for tasks in *Manage Tasks* and *Task Inbox* using the *Advanced* filters option. You can create a special filter that will provide domain specific columns to be added to a task list and the task variables are enabled as columns. Once you create and apply the special filter, you can view the task variables of the task as columns using the *Show/hide columns* option.
```

Comment:
I shortened this a little bit, and I think it is clearer for me, what do you think?

You can use the Advanced filters option in {CENTRAL} to create special filters for tasks in *Manage Tasks* and *Task Inbox*. When you create a special filter with the task defined, you can view the task variables of the task as columns using *Show/hide columns*.

Reply:
The shortened para indeed reads better.